packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts

```diff
@@ -280,3 +292,17 @@ function parseMessagesFromXml(pDoc: XMLDocument): MessageData[] {
 
   return messagesData
 }
+
+export async function getEndpointStatistics(node: MBeanNode) {
+  let stats: EndpointStatistics[] = []
+  const registry = getDefaultRuntimeEndpointRegistry(node)
+  if (registry) {
+    if (registry.objectName) {
+      const res = await jolokiaService.execute(registry.objectName ?? '', 'endpointStatistics()')
+      stats = Object.values(res as { [key: string]: EndpointStatistics })
+    } else {
+      log.error('Error with the retrieving registry')
+    }
```

Comment:
so, now the code just ignores if `registry` is absent. Is it what we expect?

Also, now you already guard with `if (regitry.objectName)` so `registry.objectName ?? ''` doesn't make sense as you can assume it's present.